client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -84,14 +93,27 @@ const BrewRenderer = createClass({
 	},
 
 	handleScroll : function(e){
-		const target = e.target;
+		let currentPage = 1;
+		if(window) {
+			const pageCollection = window.frames['BrewRenderer'].contentDocument.getElementsByClassName('page');
+			const brewRendererHeight = window.frames['BrewRenderer'].contentDocument.getElementsByClassName('brewRenderer').item(0).getBoundingClientRect().height;
+
+			for (const page of pageCollection) {
+				if(page.getBoundingClientRect().bottom > (brewRendererHeight / 2)) {
+					currentPage = parseInt(page.id.slice(1)) || 1;
+					break;
+				}
+			}
+		}
```

Comment:
This seems a lot more complicated to do essentially the same thing as before. What is the difference here? Is this just to handle different-sized pages?

Reply:
This is a copy/paste of the page detection from the Brew movement function; I needed a more robust system once I realized that the new rendering method broke the page info panel.
I can absolutely swap out to `target` to simplify the logic, or see if this can be reworked to be simpler.